hs/src/Reach/Connector/ETH_Solidity.hs

```diff
@@ -57,7 +57,9 @@ conName' :: T.Text
 conName' = "ETH"
 
 conCons' :: DLConstant -> DLLiteral
-conCons' DLC_UInt_max = DLL_Int sb $ 2 ^ (256 :: Integer) - 1
+conCons' = \case
+  DLC_UInt_max  -> DLL_Int sb $ 2 ^ (256 :: Integer) - 1
+  DLC_Zero_addr -> impossible "ETH conCons': zero_addr"
```

Comment:
See the comment about, but in terms of code generation it is possible... `address(0)`